src/plugins/summaryWidget/plugin.js

```diff
@@ -85,16 +84,7 @@ define([
 
         return function install(openmct) {
             openmct.types.addType('summary-widget', widgetType);
-            openmct.legacyExtension('policies', {
-                category: 'composition',
-                implementation: SummaryWidgetsCompositionPolicy,
-                depends: ['openmct']
-            });
-            openmct.legacyExtension('policies', {
-                category: 'view',
-                implementation: SummaryWidgetViewPolicy,
-                depends: ['openmct']
-            });
+            openmct.composition.addPolicy(new SummaryWidgetsCompositionPolicy(openmct).allow);

```

Comment:
Is `new` needed here?

Reply:
Ah, I was reading it wrong.

I think what you'll need is:
```
let compositionPolicy = new SummaryWidgetsCompositionPolicy(openmct));
openmct.composition.addPolicy(compositionPolicy.allow.bind(compositionPolicy));
```

Reply:
I'll update that, thank you.

Reply:
Should the condition widget be updated to be consistent with that approach? https://github.com/nasa/openmct/blob/master/src/plugins/condition/plugin.js#L62

Reply:
Probably a good idea. I don't really understand why it's been done that way.